setup/debian-based.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```

Comment:
it's almost the same as fedora-based.sh, would be better to merge to avoid code duplicity